### ISSUE TITLE
Passing additional options to connect-waterline

### ISSUE THREAD

**Issue by ricick** [balderdashy/sails-docs#602]
I can't see any information on how to pass additional options such as autoRemove and autoRemoveInterval to connect-waterline from the session config in the docs.

I'm working with a mongo host that doesn't support ttl indexing, so need to tell connect to remove expired sessions manually.

**Comment by mikermcneil** [comment]
Hey @ricick- I've never actually used that configuration myself, so I wouldn't be a good candidate to write it up.  That said, I was actually just in that file and cleaned things up a bit-- it's still missing a clear explanation on how to use arbitrary `connect-*` session adapters (I just haven't had time to sit down, refresh my memory on how it works, and write some semi-eloquent documentation on the subject).

However, here's how I _think_ it works (fair warning: this is untested, offhand, from memory):

> All properties in the `session` configuration dictionary [get passed in to the custom session adapter](https://github.com/balderdashy/sails/blob/master/lib/hooks/session/index.js#L160).  So any extra options you want to send along, you can just include as arbitrary extra keys in your `config/session.js` file.  In order to get `connect-waterline` working, you just use the string name of that adapter (here's where we expand shorthand notation for redis and mongo: https://github.com/balderdashy/sails/blob/master/lib/hooks/session/index.js#L93) and it is required automatically.

If you or anyone else reading this is currently using a connect session store other than Mongo or Redis in their app, would you mind taking that for a spin with your setup and the latest stable version of Sails (v0.11.4)? Thanks!

**Comment by mikermcneil** [comment]
@ricick just wanted to verify that this should definitely work, so if you're still having issues, please let me know.  Thanks!

**Comment by ricick** [comment]
Hey Mike. I'm away at the moment but i'll definately check this out when I get back. Cheers.

**Comment by sailsbot** [comment]
Thanks for posting, @ricick.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
